.github/workflows/release.yml

```diff
@@ -25,6 +25,21 @@ jobs:
         id: changesets
         uses: changesets/action@v1
         with:
+          # This expects you to have a script called release which does a build for your packages and calls changeset publish
+          publish: npm run release
           version: npm run version
+          title: "Release Plugin"
+          commit: "Release Plugin"
+          
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Upload release artifact
+        if: steps.changesets.outputs.published == 'true'
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: "@wpengine/wp-graphql-content-blocks.zip"
+          asset_name: wp-graphql-content-blocks.zip
```

Comment:
It could be helpful to add the version directory to the filename to help identify the download without needing to unzip.

Reply:
Good call! I've updated the asset name to be formated like `wp-graphql-content-blocks.0.1.1.zip` to replicate how Wordpress does it on the plugins repo:

https://downloads.wordpress.org/plugin/faustwp.0.8.1.zip